en/python_introduction/README.md

```diff
@@ -252,7 +252,11 @@ This shows that you just created an empty dictionary. Hurray!
 
 Now, try writing the following command (try replacing your own information too):
 
-    >>> participant = {'name': 'Ola', 'country': 'Poland', 'favorite_numbers': [7, 42, 92]}
```

Comment:
But whitespace is so tied to Python we can't really escape it. We should rather explain it than hide it away. Later on if/else is introduces that uses whitespace for blocks